build.gradle

```diff
@@ -75,7 +75,7 @@ buildscript {
       zxing               : '3.3.3',
       paging              : '2.1.2',
       appcompat           : '1.2.0-beta01',
-      navigation          : '2.2.2',
+      navigation          : '2.3.0',
```

Comment:
Did we check the release notes to see if there are any breaking changes? Since this is going to be cherry-picked onto a release branch, I'm a little wary of introducing possible behavioural changes.

Reply:
Yes, I have tested it after updating the version as well. There are a bunch of bug fixes and some new features, but if you aren't comfortable with making this change right now, I can drop this for now and bump it for next release. WDYT?

https://developer.android.com/jetpack/androidx/releases/navigation

Reply:
If you've tested it, it's fine. Just wanted to confirm.

Reply:
Can you verify it on your device as well, please?